presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestMergeIdenticalWindows.java

```diff
@@ -317,6 +318,98 @@ public void testNotMergeAcrossJoinBranches()
                                                 anyTree())))));
     }
 
+    @Test
+    public void testNotMergeDifferentPartition()
+    {
+        @Language("SQL") String sql = "select " +
+                "sum(extendedprice) over (partition by suppkey order by orderkey rows between unbounded preceding and current row) sum_extendedprice_A, " +
+                "sum(quantity) over (partition by quantity order by orderkey rows between unbounded preceding and current row) sum_quantity_C " +
+                "from lineitem";
+
+        Window windowC = new Window(
+                ImmutableList.of(new SymbolReference("quantity")),
+                ImmutableList.of(new SortItem(new SymbolReference("orderkey"), SortItem.Ordering.ASCENDING, SortItem.NullOrdering.UNDEFINED)),
+                Optional.of(commonFrame));
+
+        assertUnitPlan(sql,
+                anyTree(
+                        window(ImmutableList.of(
+                                functionCall("sum", windowC, false, "quantity")),
+                                window(ImmutableList.of(
+                                        functionCall("sum", windowA, false, "extendedprice")),
+                                        anyNot(WindowNode.class)))));
+    }
+
+    @Test
+    public void testNotMergeDifferentOrderBy()
+    {
+        @Language("SQL") String sql = "select " +
+                "sum(extendedprice) over (partition by suppkey order by orderkey rows between unbounded preceding and current row) sum_extendedprice_A, " +
```

Comment:
Previous code (`TestMergeIdenticalWindows`) already in master had them lower-cased, so for unification we had new tests in the same code style.
But as per your request I've added 56af9e5 with upper-casing SQL keywords in all tests (old and new), such as we have one reformatting commit separate from logical changes.